src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Count.cs

```diff
@@ -23,7 +23,7 @@ internal sealed class CountFunction : FunctionWithTableInput
         public override bool IsSelfContained => true;
 
         public CountFunction()
-            : base("Count", TexlStrings.AboutCount, FunctionCategories.Table | FunctionCategories.MathAndStat, DType.Number, 0, 1, 1, DType.EmptyTable)
+            : base("Count", TexlStrings.AboutCount, FunctionCategories.Table | FunctionCategories.MathAndStat, DType.Unknown, 0, 1, 1, DType.EmptyTable)
```

Comment:
Just double checking impact of setting this to .Unknown.

Reply:
This is what we are thinking we should revert.  I changed these to .Unknown because it is unknown (could be either Decimal or Number) and CheckTypes will determine.  This change appears to break PA Client and is making it hard for Carlos to integrate.  The interpreter doesn't appear to care either way.  So, we are in the process of changing these back, if it works, for PA Client.  Stay tuned...

Reply:
The change to DType.Number appears to have worked.  We are moving forward with having all native Decimal functions return DType.Number.  When we add Decimal to PA Client, we can revisit this approach.